src/main/scala/codacy/codenarc/CodeNarc.scala

```diff
@@ -35,7 +35,7 @@ object CodeNarc extends Tool {
 
   private def paramValueAsString(paramValue: JsValue): String =
     paramValue match {
-      case JsString(value) => s"'$value'"
+      case jsonEncodedString @ JsString(_) => s"'${jsonEncodedString.toString.stripPrefix("\"").stripSuffix("\"")}'"
```

Comment:
i had to do the strips because of we want to get rid of double quotes to keep the backslashes as it is